### PR TITLE
Added forseti db user and password.

### PR DESCRIPTION
**Pull request by joecheuk**
Resolves: https://github.com/forseti-security/forseti-security/issues/3217
Relates: https://github.com/forseti-security/forseti-security/pull/3266

### CODE REVIEW

File: modules/server/main.tf [hunk 1]
```diff
@@ -569,11 +571,12 @@ resource "google_sql_database" "forseti-db" {
   instance = google_sql_database_instance.master.name
 }
 
-resource "google_sql_user" "root" {
-  name     = "root"
+resource "google_sql_user" "forseti_db_user" {
```
Comment (gkowalski-google) [inline modules/server/main.tf]: @kevensen Depending on when this gets merged, you may need to update your PR to move this to the cloudsql module.

File: modules/server/templates/scripts/forseti-server/forseti_env.sh.tpl [hunk 2]
```diff
@@ -1,3 +1,5 @@
 export SQL_PORT=${cloudsql_db_port}
+export SQL_DB_USER=${cloudsql_db_user}
+export SQL_DB_PASSWORD=${cloudsql_db_password}
```
Comment (gkowalski-google) [inline modules/server/templates/scripts/forseti-server/forseti_env.sh.tpl]: Should we consider encrypting the password with KMS?

**Comment by joecheuk** [comment]
Re-opening a new PR due to the sql module being pull out from the forseti server module.

**Comment by kevensen** [comment]
@joecheuk Is this going into 2.23 or 2.24?

**Pull request closed without merging**